community/modules/scheduler/schedmd-slurm-gcp-v5-controller/README.md

```diff
@@ -82,7 +85,19 @@ This option has some additional requirements:
 * The project in your gcloud config must match the project the cluster is being
   deployed onto due to a known issue with the reconfigure scripts. To set your
   default config project, run the following command:
-  `gcloud config set core/<<PROJECT ID>>`
+
+  ```bash
+  gcloud config set core/<<PROJECT ID>>
```

Comment:
do you mean `gcloud config set core/project <<PROJECT_ID>>`

I think an alternative might be ``gcloud config set project <<PROJECT_ID>>`

Reply:
Good catch! Both your suggestions worked for me. I will stick with `core/project` as that was the original intention.